source/document-types.html.md

```diff
@@ -3,26 +3,31 @@ layout: document_type_layout
 title: Document types on GOV.UK
 ---
 
-The document type describes a type of page that can exist on GOV.UK. This is different to the [content schemas][] which describe the underlying structure of a piece of content.
+A _document type_ describes a category of pages that can exist on GOV.UK. Document types are different from [content schemas], which define the structure of a piece of content.
 
-The document types included in these docs are pulled in automatically based on data in the [govuk_document_types repo][].
+GOV.UK Developer Docs automatically generates documentation about document types based on data in the [govuk_document_types repo].
 
-## Analysing document types
+## Relationship between document type and content schema
 
-You can also [download this data as CSV](/document-types.csv). To view the current data in Google Spreadsheets, copy the following formula into a new spreadsheet:
+The mapping between GOV.UK document types and content schemas is many-to-many. A document type can correspond to one or more content schemas. A content schema can permit one or more document types for content items that conform to that schema.
```

Comment:
I like this - explains it clearly and explicitly :+1:

Reply:
Thanks! 😄